### ISSUE TITLE
 b.cesm3_cam058_mom_e.B1850WscMOM.ne30_L58_t061.camdev_cice5.023

### ISSUE THREAD

**Issue by cecilehannay** [NCAR/amwg_dev#158]
**Description:**
Same as [022](https://github.com/NCAR/amwg_dev/issues/157) but changing the vertical from L32 -> L58 
This run uses the same tuning as 022. No additional tuning has been added to adjust RESTOM for L58. 

It means this run uses:
```
clubb_gamma_coef   =  0.28
clubb_c14          = 2.2D0
clubb_l_trapezoidal_rule_zm          =  .true.
clubb_l_trapezoidal_rule_zt          =  .true.
```
This is different from the 009 tuning:
```
clubb_gamma_coef     =  0.270   
clubb_c14            =  1.6D0  
clubb_l_trapezoidal_rule_zm          =  .false.
clubb_l_trapezoidal_rule_zt          =  .false. 
```

**Case directory:**
Locally (if still available):
/glade/p/cesmdata/cseg/runs/cesm2_0/ b.cesm3_cam058_mom_e.B1850WscMOM.ne30_L58_t061.camdev_cice5.023

On github:
https://github.com/NCAR/amwg_dev/tree/ b.cesm3_cam058_mom_e.B1850WscMOM.ne30_L58_t061.camdev_cice5.023

**Sandbox:**
Locally (if still available):
/glade/work/hannay/cesm_tags/cesm3_cam6_3_058_MOM_e

On github:
hash: 8f70c08

**Diagnostics:**
AMWG diags (if available)
[https://webext.cgd.ucar.edu/BWsc1850MOM/ b.cesm3_cam058_mom_e.B1850WscMOM.ne30_L58_t061.camdev_cice5.023/atm/](https://webext.cgd.ucar.edu/BWsc1850MOM/ b.cesm3_cam058_mom_e.B1850WscMOM.ne30_L58_t061.camdev_cice5.023/atm/)

**Contacts:**
@cecilehannay, @gustavo-marques, @JulioTBacmeister, @fobryan3

**Comment by megandevlan** [comment]
My hunch is that the tuning for L58 will be important - I'm not sure what the RESTOM looks like, but the surface fluxes by year 12 seem a little strange. LHFLX is growing and up to 90.5 W/m2, and the SWCF looks too weak (-42 W/m2 rather than -45 W/m2). Hard to say how the Lab Sea will respond though, right now the fluxes don't look crazy there but it'll be easier to tell once the run has more years. 

Is the plan to run a tuned version of this case as well? 

<img width="1026" alt="Screen Shot 2022-08-12 at 9 51 34 AM" src="https://user-images.githubusercontent.com/42625059/184394185-a4d0ca27-911a-465a-beaf-445809985ae4.png">

**Comment by fobryan3** [comment]
How does 023 differ from 009? CIE5 vs. CICE6 and CLUBB tuning?

**Comment by cecilehannay** [comment]
@Meg: Yes, we are planning to do some tuning. This is the same strategy as we moved from FV->SE. In the first SE run (017), we just change the dycore without doing any tuning, then we retuned in 018b and 19b.
We are using the same strategy here. First, changing L32->L58. Second, do some tuning.

@fobryan3: This is correct. The main differences between 009 and 023 are clubb tuning and cice5. There is also a slight change in the sandbox.

**Comment by adamrher** [comment]
I don't recall such a large reduction in SWCF in our F-case L32-L58 experiments. I recall a small reduction, but not 3-4 W/m2.

Lets take a look at the diagnostics when they are ready, but I don't think there's any getting away from having to reduce clubb skewness (via clubb_gamma or c8) to thicken the clouds back up.

**Comment by JulioTBacmeister** [comment]
We're also using the CLUBB trapezoidal rule interpolation in 023 as we've been doing in all the L32 runs. The effect at L58 is probably not as large as it is in L32, where it has a large impact. But we need to decide what to do.

**Comment by adamrher** [comment]
I personally am not a fan of the how the trapezoidal rules are implemented in clubb. They introduce super-saturation because the trapezoidal interpolation is performed **after** the saturation adjustment. I vote for turning them off, especially b/c its less consequential at L58. But perhaps I shouldn't be such a purest at this stage of the work.

Bigger picture, it's going to take more than the trap rules to get us another 3-4 W/m2 in L58.

**Comment by cecilehannay** [comment]
In L32, the trapezoidal rules changed RESTOM by 2 W/m2.  Do you have a sense how it translates with L58.

**Comment by adamrher** [comment]
We did look at L58 sens to the trap rules a while back. I dug it up here: https://github.com/NCAR/amwg_dev/issues/28. Looks like its about 1 W/m2 cooler w/ the trap rules off, in L58.

**Comment by JulioTBacmeister** [comment]
I think our next coupled run should turn-off the trapezoidal rule and we should return to the issue of tuning via c8.  I think it's pretty clear from 023 (see figure below - thick blue line with circles) that we CAN get an ice-free Lab Sea with L58+camdev+no-leak-topo if we are willing to start with a RESTOM of ~ +1.5 Wm-2 and a global TS > 289.0K.   We need to think about how the LabSea is connected to these quantities - does SST and cloud cover in the southeast Pacific/ Peruvian StCu deck really matter? Probably not, but broader N Atlantic and maybe Arctic surface fluxes probably do.

![image](https://user-images.githubusercontent.com/9221710/184503782-e2882868-9d97-4af6-9a9a-d981126a98e0.png)

**Comment by JulioTBacmeister** [comment]
Attached are some comparisons of Years 009 #136 (which froze) and 023 #158 (which looks like it won't).  Note both thse runs are L58+camdev+no_leak_topo (009 uses CICE6, 023 uses CICE5). From the plots below it looks like to bring down RESTOM in 023 without freezing the Lab Sea we should try to thicken subtropical low-clouds without touching mid-high latitude low clouds

Maps of Year 1-10:

TS and FSNT-FLNT
![image](https://user-images.githubusercontent.com/9221710/184505851-816b904d-806a-4abe-ad94-5a138c262124.png)

TS and LWCF and SWCF
![image](https://user-images.githubusercontent.com/9221710/184505871-36b24b46-672c-413a-9937-60e26233d148.png)

TS and LHFLX and SHFLX
![image](https://user-images.githubusercontent.com/9221710/184505888-f28d1741-f295-4e7e-922c-e6c2cd587724.png)

**Comment by adamrher** [comment]
ok so I'm thinking just a little nudge on the clubb parameters. c8 = 4.2->4.4 and turn off the trap rules. You may recall from our 19b experiment, we set c8=4.2->4.6, and that seemed to be a little overkill. That was in L32, so there's an important difference there.

The trap rules tend to target the subtropical marine Sc. c8 should target that same region. But it would be nice to see the change in SWCF for 17 vs 19b, to gauge the spatial pattern of changing c8=4.2->4.6 in that run.

**Comment by adamrher** [comment]
Here's the SWCF sensitivity between runs 17, 18b and 19b. Note these are all L32 runs. This is to help us understand the sensitivity to changing clubb parameters, since it seems like we will have to do this as run 23 is too warm.

Change in SWCF, 18b-17. This illustrates the impact of changing clubb_gamma=0.298->0.280

![temp_SWCF 18b](https://user-images.githubusercontent.com/29961476/184518220-e1955df4-11d6-4ce5-b94d-012bd514f5fe.png)

Change in SWCF, 19b-17. This illustrates the impact of changing clubb_c8=4.2->4.6

![temp_SWCF 19b](https://user-images.githubusercontent.com/29961476/184518222-7f24457d-f49a-4e50-938c-8f3b07ab3a8d.png)

The spatial patterns due to changing clubb_c8 and clubb_gamma are remarkably similar. Since the change from clubb_c8=4.2->4.6 is globally -2.4 W/m2, it would be reasonable to expect changing c8=4.2->4.4 would be close to -1.2 W/m2. 

The trapezoidal rules tuned off might gives us another -1 W/m2, but I'm less confident in this estimate, and suspect it will be less.
